packages/components/src/Field/Field.tsx

```diff
@@ -1,49 +1,52 @@
 import React from 'react';
-import { Input, Label, ValidationMessage } from '@marigold/components';
-import { useStyles, system } from '@marigold/system';
+
 import { Exclamation, Required } from '@marigold/icons';
-import b2bTheme from '@marigold/theme-b2b';
+import { useStyles } from '@marigold/system';
+import { ComponentProps } from '@marigold/types';
+
+import { Input } from '../Input';
+import { Label } from '../Label';
+import { ValidationMessage } from '../ValidationMessage';
 
-type FieldProps = {
+export type FieldProps = {
```

Comment:
Why this export?

Reply:
always export props ... I noticed that there are other components that do not do that too.